<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMagicFish</class>
 <widget class="QWidget" name="frmMagicFish">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="MagicFish" name="magicFish" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QSlider" name="horizontalSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ColorComboBox" name="comboBox"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MagicFish</class>
   <extends>QWidget</extends>
   <header location="global">magicfish.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">colorcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
